packages/validator/src/cli-validator/run-validator.js

```diff
@@ -136,14 +127,20 @@ async function runValidator(cliArgs, parseOptions = {}) {
     file => !filesToValidate.includes(file)
   );
   nonExistentFiles.forEach(file => {
-    logger.warn(
-      chalk.yellow('[Warning]') + ` Skipping non-existent file: ${file}`
-    );
+    logger.warn(`Skipping non-existent file: ${file}`);
   });
 
   // If no passed in files are valid, exit the program.
   if (!filesToValidate.length) {
-    logError(chalk, 'No files to validate.');
+    logger.error('No files to validate.');
+    return Promise.reject(2);
+  }
+
+  // If multiple files were specified and JSON output is requested, exit with an error.
+  if (filesToValidate.length > 1 && context.config.outputFormat === 'json') {
+    logger.error(
+      'At most one file can be specified when JSON output is requested.'
+    );
```

Comment:
This may be worth mentioning in the migration guide

Reply:
done